docs/internships/outreachy/getting-started.md

```diff
@@ -0,0 +1,46 @@
+---
+id: getting-starteds
+title: Our repositories
+sidebar_label: Getting Started
+---
+
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
+1. [Talawa API github](https://github.com/PalisadoesFoundation/talawa-api)
+1. [Talawa Admin github](https://github.com/PalisadoesFoundation/talawa-admin)
+1. [Talawa Docs github](https://github.com/PalisadoesFoundation/talawa-docs).
+## Getting started with our repositories
+
+In our github repositories we have a few files that include guidelines and rules to guide applicants on how to contribute and the rules to follow in other to open a acceptable pull request and give a meaningful contribution.
+
+1. **CODE_OF_CONDUCT.md**
```

Comment:
1. Create an H2 title `Important Files` then...
1. Make each of these bullets H3 titles under that